src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,45 @@
-import React from 'react';
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 import './FindMovie.scss';
 
-export const FindMovie: React.FC = () => {
+type Props = {
+  onFind: (arg0: string) => void;
+  movie: Movie | null;
+  onAdd: () => void;
+  isLoading: boolean
+};
+
+export const FindMovie: React.FC<Props> = ({
```

Comment:
nb: destruct FC from react

Reply:
no, i prefere this variant

Reply:
Why do you prefer this variant? :)
It's ok, but what is the reason? )

Reply:
I believe that when i use syntax React.FC, it is much more readable and easy to understand where this variable comes from. The same goes with destructuring props. I get a lot of comments from mentors that i need to destructure them but i believe that when i use props.children, the code becomes much more readable for me and others

Reply:
You may change your mind when you will work on a huge project :)
But ok )

P. S. Destructuring is a trending best practice. So you should remember this.